contributors/devel/sig-node/cherry-picks.md

```diff
@@ -24,7 +24,7 @@ cherry-picked to previous releases must be [triaged], prioritized, LGTM'd, and
 approved by the deadline.
 
 [patch releases]: https://github.com/kubernetes/sig-release/blob/master/release-engineering/versioning.md#patch-releases
-[target dates]: https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md#upcoming-monthly-releases
+[target dates]: https://github.com/kubernetes/website/blob/main/content/en/releases/patch-releases.md#upcoming-monthly-releases
```

Comment:
this link should be: https://kubernetes.io/releases/patch-releases/#upcoming-monthly-releases